test/support/supportedExtensions.ts

```diff
@@ -0,0 +1,277 @@
+/* tslint:disable max-line-length */
```

Comment:
As much as I would like to agree with, I don't. If the node structure of the supported icons changes so should the test. The tests will be useless if they use a redundant node entry structure.